### PR TITLE
Update spec for python 3.11 build

### PR DESCRIPTION
**Pull request by smarlowucf**
Update ci python testing versions

### CODE REVIEW

**Review by apozsuse — APPROVED** [review]
LGTM

File: python3-gceimgutils.spec [hunk 1]
```diff
@@ -17,25 +17,32 @@
 
 
 %define upstream_name gceimgutils
+%define python python
```
Comment (rjschwei) [inline python3-gceimgutils.spec]: Also needs alternative definitions, see https://github.com/SUSE-Enceladus/ec2imgutils/blob/master/python-ec2imgutils.spec#L20

File: python3-gceimgutils.spec [hunk 2]
```diff
@@ -60,8 +67,9 @@ gzip %{buildroot}/%{_mandir}/man1/*
 %doc README.md
 %license LICENSE
 %{_mandir}/man*/*
-%dir %{python3_sitelib}/gceimgutils
-%{python3_sitelib}/*
+%dir %{python_sitelib}/gceimgutils
+%{python_sitelib}/*
```
Comment (rjschwei) [inline python3-gceimgutils.spec]: the "binary" needs an alternative setup, see https://github.com/SUSE-Enceladus/ec2imgutils/blob/master/python-ec2imgutils.spec#L146

**Review by rjschwei — CHANGES_REQUESTED** [review]
Also needs %pre,%post and %postun setup, see https://github.com/SUSE-Enceladus/ec2imgutils/blob/master/python-ec2imgutils.spec#L119

**Review by rjschwei — APPROVED** [review]

**Pull request merged**